sdk/formrecognizer/test-resources.json

```diff
@@ -58,7 +34,7 @@
         },
         "trainingDataResourceId": {
             "type": "string",
-            "defaultValue": "[resourceId('TrainingData', 'Microsoft.Storage/storageAccounts', parameters('trainingDataAccount'))]"
+            "defaultValue": "[resourceId('2cd617ea-1866-46b1-90e3-fffb087ebf9b', 'TrainingData', 'Microsoft.Storage/storageAccounts', parameters('trainingDataAccount'))]"
```

Comment:
TrainingData is a fixed resource in an Test subscription, include S_ID here so the template can be deployed to arbitrary subscription.

Reply:
"include S_ID"
not sure about the impact of this, but are we still going to be able to access our static training container?

Reply:
Yes, it was always in subscription id `2cd617ea-1866-46b1-90e3-fffb087ebf9b` I'm just making sure it's more precisely addressed in the ARM template.

Reply:
ahh got it. 
Thank you for explaining!

Reply:
To clarify this will only work if the service principal we are using has access to the test subscription. If we end up using a different subscription that might not be the case.